src/components/Authentication/Login.jsx

```diff
@@ -30,13 +32,19 @@ const Login = ({ cookies }) => {
   const [password, setPassword] = useState();
   const [showPassword, setShowPassword] = useState(false);
   const [errorMessage, setErrorMessage] = useState();
+  const {
+    register,
+    handleSubmit,
+    // commented for prettier
+    // formState: { errors },
+  } = useForm();
 
   /**
    * This function handles logging in with email/password (standard log in)
    * If the user signs in successfully, they are redirected to /, otherwise they are redirected to the login screen
    * @param {Event} e
    */
-  const handleSubmit = async e => {
+  const handleFormSubmission = async data => {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'handleFormSubmission' is assigned a value but never used.